### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ericelliott**
Add static property descriptors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ The stamp descriptor properties are made available on each stamp as `stamp.compo
 * `staticProperties` - A set of static properties that will be copied by assignment to the stamp.
 * `propertyDescriptors` - A set of [object property
 descriptors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties) used for fine-grained control over object property behaviors.
+* `staticPropertyDescriptors` - A set of [object property
```
Comment (koresar) [inline README.md]: There should be no EOL after the "property" word.

File: README.md [hunk 2]
```diff
@@ -69,6 +71,7 @@ Descriptors are composed together to create new descriptors with the following r
 * `initializers` are appended: `descriptor.initializers = descriptor1.initializers.concat(descriptor2.initializers)`
 * `staticProperties` are copied by assignment: `descriptor.staticProperties = _.assign({}, descriptor1.staticProperties, descriptor2.staticProperties)`
 * `propertyDescriptors` are deep merged: `descriptor.propertyDescriptors = _.merge({}, descriptor1.propertyDescriptors, descriptor2.propertyDescriptors)`
+* `staticyPropertyDescriptors` are deep merged: `descriptor.propertyDescriptors = _.merge({}, descriptor1.propertyDescriptors, descriptor2.propertyDescriptors)`
```
Comment (koresar) [inline README.md]: Typo `staticyPropertyDescriptors` :)
Remove the `y`.

Comment (ericelliott) [inline README.md]: Good catch. =)

**Comment by ericelliott** [comment]
Thanks. =) :+1:

**Pull request merged**